packages/ag-charts-community/src/scene/shape/sector.ts

```diff
@@ -54,11 +58,23 @@ export class Sector extends Path {
                 path.arc(centerX, centerY, innerRadius, endAngle, startAngle, true);
             }
         } else {
-            path.moveTo(centerX + innerRadius * Math.cos(startAngle), centerY + innerRadius * Math.sin(startAngle));
-            path.arc(centerX, centerY, outerRadius, startAngle, endAngle);
+            const innerAngleOffset = inset !== 0 && innerRadius !== 0 ? inset / innerRadius : 0;
```

Comment:
Could you verify that the PieSeries animations are happy after these changes? (Sector rendering changes are tingling my spidey-sense)

Reply:
The radial bar works - should be fine?